lib/ansible/modules/package_facts.py

```diff
@@ -278,11 +278,11 @@ def list_installed(self):
         return self._lib.TransactionSet().dbMatch()
 
     def get_package_details(self, package):
-        return dict(name=package[self._lib.RPMTAG_NAME],
-                    version=package[self._lib.RPMTAG_VERSION],
-                    release=package[self._lib.RPMTAG_RELEASE],
-                    epoch=package[self._lib.RPMTAG_EPOCH],
-                    arch=package[self._lib.RPMTAG_ARCH],)
+        return dict(name=to_text(package[self._lib.RPMTAG_NAME]),
```

Comment:
i would use `str` at this point, to_text does not really add anything in this context